app/components/confirmationCard.tsx

```diff
@@ -0,0 +1,48 @@
+import { View, Text, StyleSheet, Image } from 'react-native';
+
+const ConfirmationCard = () => {
+    return (
+        <View style={styles.container}>
+            <Image style={{ alignSelf: 'flex-start'}} source={require("../../assets/book.png")}/>
+            <Text style={styles.text1}>Nachhilfe mit Leon</Text>
+            <Text style={styles.confirmationText}>Bestätigung</Text>
+            <Text style={styles.text3}>Leon hat uns um eine Bescheinigung für eure Lernunterstützung gebeten. Nur wenn du dies bestätigst, können</Text>
+            <Text style={styles.text4}>Angaben bestätigen</Text>
```

Comment:
If this is a button, please make it a Pressable component (I know it's a dummy but let's keep things technically correct)